ceph.spec.in

```diff
@@ -20,18 +20,20 @@
 %endif
 %bcond_with make_check
 
-%ifnarch ppc64 ppc64le s390 s390x
+%ifarch aarch64 x86_64
 %bcond_without tcmalloc
-%if ! 0%{?is_opensuse}
-%bcond_without xio
-%endif
 %bcond_without valgrind_devel
 %else
 %bcond_with tcmalloc
-%bcond_with xio
 %bcond_with valgrind_devel
 %endif
 
+%if 0%{?is_opensuse}
+%bcond_with xio
+%else
+%bcond_without xio
```

Comment:
Because it won't have any effect. Since the `%_with_xio` macro is set in the SES4 project (which is locked), and since that macro is inherited by all SES4-related projects, the conditional `%if 0%{with xio}` will always be true in SES4, unfortunately.